testapi.pm

```diff
@@ -1791,7 +1791,7 @@ sub data_url($) {
 
 =for stopwords GiB failok OpenQA WebUI
 
-  upload_logs($file [, failok => 0, timeout => 90 ]);
+  upload_logs($file [, failok => 0, timeout => 90, upname => "custom_name" ]);
 
 Upload C<$file> to OpenQA WebUI as a log file and
 return the uploaded file name. If failok is not set, a failed upload or
```

Comment:
please add an explanation of `upname` in this section as well then

Reply:
upname is a stupid paramater name.  what's wrong with using real words?

Reply:
probably , but it is not me who added this name https://github.com/asmorodskyi/os-autoinst/commit/c490d2c721f0e21a101f27ed7a671b0b0cc33961 I just use whatever was before me. But if you have better ideas I open to change to anything what you think suites better

Reply:
@okurz - explanation added

Reply:
but that upname is an internal parameter, nothing humans type